src/components/vehicles/bmwbc/vehicle.vue

```diff
@@ -26,6 +26,24 @@
     >
       <template #help> Die Fahrgestellnummer des Fahrzeugs. </template>
     </openwb-base-text-input>
+    <openwb-base-text-input
+      title="Captcha-Token"
+      required
+      :model-value="vehicle.configuration.captcha_token"
+      @update:model-value="updateConfiguration($event, 'configuration.captcha_token')"
+    >
+      <template #help> Zum erstmaligen Login z.B. nach einem Neustart ist ein <b>aktuelles Captcha-Token</b> notwendig.<br>
+                       Dazu bitte folgende Schritte durchführen:<br>
+                       1. in einem neuen Browser-Tab auf diese Seite gehen:<br>
+                          <b>https://bimmer-connected.readthedocs.io/en/latest/captcha/rest_of_world.html</b><br>
```

Comment:
Macht es nicht Sinn, hier gleich einen Link zu verwenden, den man anklicken kann? `<a href=...>` Es gibt reichlich Beispiele hier, wo ein Link im neuen Tab/Fenster geöffnet wird, z.B. der Link zum Wiki in der Menüzeile.

Reply:
Danke für den Hinweis - hatte ich mir auf die Schnelle nicht zugetraut - ich bin da nicht so fit.
Ist eingebaut  - relativ blind kopiert - funktioniert hier.